site/docs/guides/contributor/release-process/egeria-core-release.md

```diff
@@ -0,0 +1,58 @@
+
+??? success "4. Update master from `x.y-SNAPSHOT` to `x.z-SNAPSHOT`"
+    - `git checkout master`
+    - `git pull upstream master`
+    - Edit all files (command line or IDE) to replace `x.y-SNAPSHOT` with the next version, e.g. change `1.3-SNAPSHOT` to `3.1-SNAPSHOT`. Most of the changes are in `pom.xml` files, however some code and documentation also has references to our versions and all need modifying.
+    - If using an IDE like IntelliJ, make sure you have all hits by searching again as [by default only a limited number of hits are shown :material-dock-window:](https://youtrack.jetbrains.com/issue/IDEA-157855){ target=intellij }.
+    - Commit
+    - Now remove all the release notes from the `release-notes` directory other than README.md - so users will always get directed to the latest in master
+    - Commit
+    - Create a PR, have reviewed / approved and merged as usual - aim to do this as quickly as reasonable so that there is no potential for version clash
+
+??? success "5. Test, merge any remaining required changes into branch"
+    - Run appropriate tests for the release. For example, in addition to automated tests: check notebooks, run the CTS and check for compliance, check the user interface.
+    - Raise issues for any changes required as usual
+    - Note that approval is required for changes going into a release branch
+    - PR builds are run as usual; however, merge builds, Sonar, etc do not run
+    - To backport changes from `master`, first wait until the PR is merged into `master`, then use `git cherrypick -s <commithash>` to apply to `egeria-release-x.y`, then push as usual.
+    - In some cases a merge commit will need to be made using `git cherrypick -s -m 1 <commithash>`
+    - If code has diverged significantly a manual recode may be easiest
+
+??? success "6. Update branch's release version from `x.y-SNAPSHOT` to `x.y`"
+    - Aim to make this change when the code appears to be ready to ship apart from final tests in order to avoid version confusion
+    - `git checkout egeria-release-x.y`
+    - `git pull upstream egeria-release-x.y`
+    - Edit all files (command line or IDE) to replace `x.y-SNAPSHOT` with `x.y`, i.e. removing the `-SNAPSHOT` designation. Most of the changes are in `pom.xml` files; however, some code and documentation also has references to our versions and all need modifying.
+    - Commit, and do not make any other changes.
+    - Create a PR, have reviewed / approved and merged as usual
+
+??? success "7. Create a release in GitHub"
+    - [Create the GitHub release :material-github:](https://github.com/odpi/egeria/releases){ target=gh }. Use `Vx.y` as the tag, and ensure the correct branch is set for the target, i.e. `egeria-release-x.y`
+    - Fill in the release notes using a title of `Release x.y` and [copy the notes from the appropriate release notes](/release-notes/overview)
+    - Artifacts will be available on Maven Central within around half a day.
```

Comment:
Use github to auto-generate notes.

However
 a) Check the 'diff' at the end of the inserted text and correct the release names
 b) manually edit the list to start at the point after the last change to the previous release (they are in order)

Reply:
@planetf1 I copied these core release notes as is. I am not sure what you are looking for here.